### DISCUSSION TITLE
Collada import

### DISCUSSION THREAD

**Discussion by merco** [hjoykim/THREE#50] [General]
I created a small converter from COLLADA format (*.dae), however there are some oddities:

- The model appears rotated on the Z axis by -90°

- There is something strange I think in the normals. The colors are not like when I import from other formats. They seem "upside down"

[https://community.khronos.org/t/coordinate-system-clarification/5810/14](url)

![immagine](https://github.com/user-attachments/assets/6dc81946-9ee6-45a0-8603-465e5e113285)

![immagine](https://github.com/user-attachments/assets/ac7f67c6-93f3-4be2-8ff7-bf5cb3f42ffd)

JS collada loader 
[ColladaLoader.zip](https://github.com/user-attachments/files/17445651/ColladaLoader.zip)

**Comment by hjoykim** [comment]
hello
I'v checked ColladaLoader.js that you've attached.
please let me know your c# loader for Collada file.

**Comment by merco** [comment]
Hi. At the moment I'm giving up the idea of ​​switching from collada format, due to other slow conversion problems.
